scripts/update_proto.sh

```diff
@@ -13,5 +13,8 @@ then
     exit 1
 fi
 
+brew upgrade protobuf
+brew upgrade swift-protobuf
```

Comment:
Could we check that `brew` is available before running these? Maybe:

```suggestion
if command -v brew >/dev/null 2>&1; then
    echo "Updating protobuf and swift-protobuf with homebrew"
    brew upgrade protobuf
    brew upgrade swift-protobuf
else
    echo "Homebrew is not installed, check protobuf and swift-protobuf versions manually"
fi
```